draft-ietf-rats-eat.md

```diff
@@ -206,7 +206,10 @@ The relying party may choose to trust, not trust or partially trust the entity.
 For example, partial trust may be allowing a monetary transaction only up to a limit.
 
 The security model and goal for attestation are unique and are not the same as for other security standards like those for server authentication, user authentication and secured messaging.
-The reader is assumed to be familiar with the goals and security model for attestation as described in {{RATS.Architecture}}.
+To give an example of one aspect of the difference, consider the association and life-cycle of key material.
+For authentication, keys are associated with a user or service and set up by actions performed by a user or an operator of a service.
+For attestation, the keys are associated with specific devices and are configured by device manufacturers.
```

Comment:
```suggestion
For attestation, the device identity keys are associated with specific devices and are configured by device manufacturers.
```

Reply:
While I know device identity is a commonly used term in some circles, neither RATS Architecture nor EAT define it or use it, so I'd prefer not to make this change.